packages/stylelint-plugin/src/util/variableUtils.ts

```diff
@@ -37,11 +42,13 @@ export const parseVariable = (node: WordNode): Variable | null => {
   const firstChar = cleanedValue[0]
   if (firstChar === "@" || firstChar === "$") {
     const name = cleanedValue.substr(1)
+    const foundRemovedToken = removedKaizenTokensByName[name]
+    const foundKaizenToken = kaizenTokensByName[name]
     return {
       name,
       nameWithPrefix: cleanedValue,
       prefix: firstChar,
-      kaizenToken: kaizenTokensByName[name],
+      kaizenToken: foundKaizenToken || foundRemovedToken,
```

Comment:
```suggestion
      // Important bit here. If a token exists in the consumers version of `@kaizen/design-tokens`, it will take precedence over ones that we know are removed.
      kaizenToken: foundKaizenToken || foundRemovedToken,
```